server/internal/adapter/gql/resolver_mutation_field.go

```diff
@@ -18,25 +18,41 @@ func ToSchemaTypeProperty(tp *gqlmodel.SchemaFieldTypePropertyInput, t gqlmodel.
 		if x == nil {
 			return nil, errors.New("invalid type property")
 		}
-		tpRes = *schema.NewFieldTypePropertyText(x.DefaultValue, x.MaxLength)
+		tp1, err := schema.NewFieldTypePropertyText(x.DefaultValue, x.MaxLength)
+		if err != nil {
+			return nil, err
+		}
+		tpRes = *tp1
 	case gqlmodel.SchemaFiledTypeTextArea:
 		x := tp.TextArea
 		if x == nil {
 			return nil, errors.New("invalid type property")
 		}
-		tpRes = *schema.NewFieldTypePropertyTextArea(x.DefaultValue, x.MaxLength)
+		tp1, err := schema.NewFieldTypePropertyTextArea(x.DefaultValue, x.MaxLength)
+		if err != nil {
+			return nil, err
+		}
+		tpRes = *tp1
```

Comment:
fix line 14: 

```go
var tpRes *schema.TypeProperty
var err error
```

and then

```suggestion
		tpRes, err = schema.NewFieldTypePropertyTextArea(x.DefaultValue, x.MaxLength)
```

Fix others also